test/e2e/btc_rewards_distribution_e2e_test.go

```diff
@@ -297,19 +297,6 @@ func (s *BtcRewardsDistribution) Test4CommitPublicRandomnessAndSealed() {
 	)
 	wg.Add(2)
 
-	go func() {
-		defer wg.Done()
-		appHash = n1.AddFinalitySignatureToBlock(
-			s.fp1BTCSK,
-			s.fp1.BtcPk,
-			s.finalityBlockHeightVoted,
-			s.fp1RandListInfo.SRList[s.finalityIdx],
-			&s.fp1RandListInfo.PRList[s.finalityIdx],
-			*s.fp1RandListInfo.ProofList[s.finalityIdx].ToProto(),
-			fmt.Sprintf("--from=%s", wFp1),
-		)
-	}()
-
 	go func() {
 		defer wg.Done()
```

Comment:
fp2 should vote first as he has lower voting power 

This was a fix for a flaky e2e test where it could happen that the block would be finalized only with fp1 vote (since he had enough vp to finalize the block by himself)